crates/ironrdp-rdpsnd-native/examples/cpal.rs

```diff
@@ -5,7 +5,9 @@ use std::thread;
 use anyhow::Context;
 use cpal::traits::StreamTrait;
 use ironrdp_rdpsnd::pdu::{AudioFormat, WaveFormat};
-use ironrdp_rdpsnd_native::cpal::make_stream;
+use ironrdp_rdpsnd_native::cpal::DecodeStream;
+#[cfg(feature = "opus")]
+use opus as _;
```

Comment:
suggestion: It’s fine to silent the warning about unused dependencies for examples. (If that’s what you got.)

Reply:
yes, although I don't understand why I get the warning here...

Reply:
Unfortunate false positives because it's a separate binary :/